argilla-server/src/argilla_server/telemetry.py

```diff
@@ -17,114 +17,102 @@
 import logging
 import platform
 import uuid
-from typing import Any, Dict, Optional
+from typing import Optional, Union
 
 from fastapi import Request
+from huggingface_hub.utils import send_telemetry
 
+from argilla_server._version import __version__
 from argilla_server.constants import DEFAULT_USERNAME
-from argilla_server.models import User
+from argilla_server.models import User, Workspace
 from argilla_server.settings import settings
 from argilla_server.utils._telemetry import (
     is_running_on_docker_container,
     server_deployment_type,
 )
 
-try:
-    from analytics import Client  # This import works only for version 2.2.0
-except (ImportError, ModuleNotFoundError):
-    # TODO: show some warning info
-    settings.enable_telemetry = False
-    Client = None
-
-
 _LOGGER = logging.getLogger(__name__)
 
 
 @dataclasses.dataclass
 class TelemetryClient:
     enable_telemetry: dataclasses.InitVar[bool] = settings.enable_telemetry
     disable_send: dataclasses.InitVar[bool] = False
-    api_key: dataclasses.InitVar[str] = settings.telemetry_key
-    host: dataclasses.InitVar[str] = "https://api.segment.io"
 
     _server_id: Optional[uuid.UUID] = dataclasses.field(init=False, default=None)
 
     @property
     def server_id(self) -> uuid.UUID:
         return self._server_id
 
-    def __post_init__(self, enable_telemetry: bool, disable_send: bool, api_key: str, host: str):
-        from argilla_server._version import __version__
-
+    def __post_init__(self):
         self._server_id = uuid.UUID(int=uuid.getnode())
         self._system_info = {
+            "server_id": self._server_id,
             "system": platform.system(),
             "machine": platform.machine(),
             "platform": platform.platform(),
             "python_version": platform.python_version(),
             "sys_version": platform.version(),
             "deployment": server_deployment_type(),
             "docker": is_running_on_docker_container(),
-            "version": __version__,
```

Comment:
@dvsrepo @frascuchon we automatically get a geo IP, but this is o a location level an not a hardware level. I did remove "python_version" since it was already being captured.